src/module.c

```diff
@@ -1135,7 +1169,105 @@ int moduleArgFlagsConvert(int flags) {
     return realflags;
 }
 
-RedisModuleCommandArg *RM_CreateCommandArg(const char* argname, RedisModuleCommandArgType type, const char *token, const char *summary, const char* since, int flags, const char *value) {
+/* Create an opaque structure representing a command argument
+ * (For manipulating the output of COMMAND)
+ *
+ * * `argname`: Name of the argument
+ * * `type`: The type of the argument.
+ *           Some types allow an argument to have sub-arguments
+ *           (REDISMODULE_ARG_TYPE_ONEOF and REDISMODULE_ARG_TYPE_BLOCK)
+ * * `key_spec_index` : If the `type` is REDISMODULE_ARG_TYPE_KEY you
+ *                      must provide the index of the key-spec associated
+ *                      with this argument. See RedisModule_AddCommandKeySpec.
+ *                      If the argument is not a key, you must specify -1.
+ * * `token`: The token preceding the argument (optional).
+ *            Example: the argument "seconds" in SET has a token "EX")
+ *            If the argument consists of only a token (Example: "NX" in SET)
+ *            the type should be REDISMODULE_ARG_TYPE_PURE_TOKEN and the
+ *            value should be NULL
```

Comment:
```suggestion
 *            token value should be NULL
```

Reply:
the sentence is correct, it is the "value" that should be NULL (if it's a pure-token there's no value/placeholder/content or whatever we decide to call it)